### ISSUE TITLE
[Feature Request - Alt/Super+Tab] Replace Ctrl +(Numbers) shortcuts with Alt+(Numbers)

### ISSUE THREAD

**Issue by senntore** [pop-os/cosmic-launcher#162]
It would be much easier on our hand, if we are to use those shortcuts.

**Comment by Kartonrealista** [comment]
I don't see it. Whether I click Super with my thumb or pointer finger it is much less comfortable to use Alt than Ctrl.

**Comment by py-crash** [comment]
I concord with @senntore here. 

If you access the launcher pressing "Alt+Tab" you need to keep pressing the Alt key, while pressing Ctrl+3 (or the Number that you need). This requires some convoluted thumb, pinky and index action. With Numbers 4,5 and 6 is even worse. However if the shortcut would be "Alt+(Number)", the finger is already pressing the Alt key, so it would be way more natural.

In the same stile, if accesing the launcher with "Super", the Alt key is next to the Super key, so it is easier that reaching for the Ctrl key

**Comment by Kartonrealista** [comment]
> If you access the launcher pressing "Alt+Tab" you need to keep pressing the Alt key, while pressing Ctrl+3 (or the Number that you need). This requires some convoluted thumb, pinky and index action. With Numbers 4,5 and 6 is even worse. However if the shortcut would be "Alt+(Number)", the finger is already pressing the Alt key, so it would be way more natural.

Not if you use Super + Tab, which is also a supported binding. Then you can't let go of Super, and have to press Alt. Maybe the right thing would be to choose between Alt+Tab and Super+Tab as a default and remove the other one.

>In the same stile, if accesing the launcher with "Super", the Alt key is next to the Super key, so it is easier that reaching for the Ctrl key

What?
![20240815_171859](https://github.com/user-attachments/assets/b857edae-65df-4067-9a45-8b29bb094505)

**Comment by py-crash** [comment]
> 
> Not if you use Super + Tab, which is also a supported binding. Then you can't let go of Super, and have to press Alt, which is inconvenient. Maybe the right thing would be to choose between Alt+Tab and Super+Tab as a default and remove the other one.
> 

That's right. I don't have an answer for that other than, probably Alt+Tab would be more common, since is a standard on Windows and other linux DEs. Another feature would be to show Tab+(Numbers) if using Tab and Super+(Numbers) if accessed using Super.

> 
> What? 

Remember that in most laptops (including those from System 76) between the Super and the Ctrl is the Fn-Key. For example in the Adder WS:

![grafik](https://github.com/user-attachments/assets/497c9336-a688-4e60-bf19-10f4301f207a)


That makes harder to change to Ctrl than Alt

**Comment by eevmanu** [comment]
For reference:

This is related to the menu opened when pressing the keyboard shortcut for **Switch between open windows** in Settings.

<img width="1120" height="514" alt="Image" src="https://github.com/user-attachments/assets/bbd70c9b-4b24-48cb-9bc3-4bb228236434" />

<img width="1511" height="636" alt="Image" src="https://github.com/user-attachments/assets/133246fa-3fc7-466c-8e1b-ccdafedd3e59" />


pd: I'm writing this message since I took me a bit to find this issue and understand the context and the benefit (+1 to this feature request)
